gdal/frmts/tiledb/tiledbdataset.cpp

```diff
@@ -696,24 +722,51 @@ CPLErr TileDBDataset::TryLoadCachedXML( char ** /*papszSiblingFiles*/, bool bRel
         {
             CPLErrorHandlerPusher oQuietError( CPLQuietErrorHandler );
 
-            if ( vfs.is_file( psPam->pszPamFilename ) )
+#if TILEDB_VERSION_MAJOR >= 1 && TILEDB_VERSION_MINOR >= 7
+            if ( bReload )
             {
-                if ( bReload )
+                tiledb_datatype_t v_type = TILEDB_UINT8; // CPLSerializeXMLTree returns char*
+                const void* v_r;
```

Comment:
perhaps initialize v_r and v_num so we are sure they are always at well defined values ?

Reply:
+1